autoload/ale/handlers/cairo.vim

```diff
@@ -0,0 +1,41 @@
+" Author: 0xhyoga <0xhyoga@gmx.com>,
+" Description: This file implements handlers specific to Cairo
+"
+function! ale#handlers#cairo#HandleCairoErrors(buffer, lines) abort
```

Comment:
If the function isn't generally useful other than for `scarb`  you could put this in the `scarb.vim` file. Either way is good.

Reply:
I'd rather have it separate as it could be useful for future Cairo/Scarb tools. Could we leave this here?

Reply:
Yeah.